code/ui/manager/src/components/layout/Layout.tsx

```diff
@@ -171,9 +171,22 @@ export const Layout = ({ managerLayoutState, setManagerLayoutState, ...slots }:
 
 const LayoutContainer = styled.div<LayoutState>(
   ({ navSize, rightPanelWidth, bottomPanelHeight, viewMode, panelPosition }) => {
+    // To support older browsers we need to set the layout height using JS
+    // In the future we can use the CSS `height: 100dvh` instead
+    const setHeight = () => {
+      const currentHeight = window.innerHeight;
+      document.documentElement.style.setProperty('--vh', `${currentHeight}px`);
+    };
+
+    useEffect(() => {
+      setHeight();
+      window.addEventListener('resize', setHeight);
+      return () => window.removeEventListener('resize', setHeight);
+    }, []);
+
     return {
       width: '100%',
-      height: '100svh', // We are using svh to use the minimum space on mobile
+      height: 'var(--vh, 100vh)',
```

Comment:
Emotion supports array values for fallbacks:
https://emotion.sh/docs/object-styles#fallbacks

```suggestion
      height: ['100dvh', '100vh],
```

IIUC, the bug that requires `100dvh` only happens on `iOS` and that supports the CSS feature?

Reply:
This is more about supporting old browser versions, and we haven't yet decided which versions to support in SB 8.0. https://caniuse.com/viewport-unit-variants

The fallback syntax looks interesting, I can't find any info on how this works? Is it a browser feature, or does Emotion do something clever with a list of CSS features per user agent?

based on the docs you linked, I think the syntax is the other way around in your suggestion.

Reply:
Oh that's interesting. I didn't know Emotion had fallbacks. I'll try it out now.

Reply:
Having the same property twice on a single class is a CSS feature.
CSS is fault-tolerant.

Emotion allows you to use this feature with the array syntax.

True, the exact implementation might have to be reversed from the code suggestion I added.